### PR TITLE
Comment fix & tiny sha1 lookup refactoring.

### PR DESCRIPTION
**Pull request by mstyura**

### CODE REVIEW

File: src/main/java/org/jitsi/videobridge/DtlsTransport.java [hunk 1]
```diff
@@ -194,9 +195,13 @@ else if (!StringUtils.isNullOrEmpty(setup))
         {
             dtlsStack.setRemoteFingerprints(remoteFingerprints);
 
-            String hash = remoteFingerprints.entrySet().iterator().next().getKey();
+            final boolean hasSha1Hash = remoteFingerprints
+                .keySet()
+                .stream()
+                .anyMatch(hash -> hash.equalsIgnoreCase("sha1"));
```
Comment (damencho) [inline src/main/java/org/jitsi/videobridge/DtlsTransport.java]: It used to check for 'sha-1' string and now this is changed to 'sha1', is this intentional?

Comment (mstyura) [reply src/main/java/org/jitsi/videobridge/DtlsTransport.java]: Thanks a lot @damencho, nice catch! It was completely accidental change.

Comment (JonathanLennox) [inline src/main/java/org/jitsi/videobridge/DtlsTransport.java]: To be clear, this changes from checking whether the first fingerprint is sha-1, to checking whether any fingerprint is sha-1?

That's probably the right thing to do, but I wanted to make sure it was intentional.

Comment (mstyura) [reply src/main/java/org/jitsi/videobridge/DtlsTransport.java]: @JonathanLennox yes, it was intentional change. I understand that previous implementation was slightly different, but I suppose it actually meant to check presence of sha-1. BTW, I don't know if it make sense to have more than one fingerprint actually, I suppose there is only 0 or 1 such attribute possible.

Comment (JonathanLennox) [reply src/main/java/org/jitsi/videobridge/DtlsTransport.java]: Yes, you can have multiple fingerprints -- see [RFC 8122](https://tools.ietf.org/html/rfc8122).

The reason is that either you might support multiple hash functions (e.g. both sha-1 and sha-256) or you might not yet know at offer time which certificate you will be using.

Comment (mstyura) [reply src/main/java/org/jitsi/videobridge/DtlsTransport.java]: @JonathanLennox as one of the authors listed in this RFC, you are definitely know better how it supposed to work :) Let me know if this implementation comply to the spec from your point of you.

Comment (JonathanLennox) [reply src/main/java/org/jitsi/videobridge/DtlsTransport.java]: This part seems right.

(A quick inspection indicates that `DtlsStack.kt` isn't correctly handling multiple fingerprints, but that's a separate issue.)

Comment (mstyura) [reply src/main/java/org/jitsi/videobridge/DtlsTransport.java]: @JonathanLennox could you please then approve and merge? Or I need an approval from Brian or Boris as well?

Comment (JonathanLennox) [reply src/main/java/org/jitsi/videobridge/DtlsTransport.java]: It LGTM, but I don't yet have approval rights on the videobridge repo.  We'll need @bbaldino or @bgrozev.

**Pull request merged**